sympy/physics/mechanics/_actuator.py

```diff
@@ -0,0 +1,243 @@
+        required so that these can be easily concatenated with other items in
+        the list of loads and passed to ``KanesMethod.kanes_equations``. These
+        loads are also in the correct form to also be passed to the other
+        equations of motion method classes, e.g. ``LagrangesMethod``.
+
+        Examples
+        ========
+
+        The below example shows how to generate the loads produced by a force
+        actuator that follows a linear pathway. In this example we'll assume
+        that the force actuator is being used to model a simple linear spring.
+        First, create a linear pathway between two points separated by the
+        coordinate ``q`` in the ``x`` direction of the global frame ``N``.
+
+        >>> from sympy.physics.mechanics import Point, ReferenceFrame
+        >>> from sympy.physics.mechanics._pathway import LinearPathway
+        >>> from sympy.physics.vector import dynamicsymbols
+        >>> q = dynamicsymbols('q')
+        >>> N = ReferenceFrame('N')
+        >>> pA, pB = Point('pA'), Point('pB')
+        >>> pB.set_pos(pA, q * N.x)
+        >>> pathway = LinearPathway(pA, pB)
+
+        Now create a symbol ``k`` to describe the spring's stiffness and
+        instantiate a force actuator that produces a (contractile) force
+        proportional to both the spring's stiffness and the pathway's length.
+
+        >>> from sympy import Symbol
+        >>> from sympy.physics.mechanics._actuator import ForceActuator
+        >>> stiffness = Symbol('k')
+        >>> spring_force = stiffness * pathway.length
+        >>> spring = ForceActuator(spring_force, pathway)
+
+        The forces produced by the spring can be generated in the list of loads
+        form that ``KanesMethod`` (and other equations of motion methods)
+        requires by calling the ``compute_loads`` method.
+
+        >>> spring.compute_loads()
+        [(pA, k*q(t)*N.x), (pB, - k*q(t)*N.x)]
+
+        A simple linear damper can be modeled in a similar way. Create another
+        symbol ``c`` to describe the dampers damping coefficient. This time
+        instantiate a force actuator that produces a force proportional to both
+        the damper's damping coefficient and the pathway's shortening velocity.
+        Note that the damping force is negative as it acts in the opposite
+        direction to which the damper is changing in length.
+
+        >>> damping_coefficient = Symbol('c')
+        >>> damping_force = -damping_coefficient * pathway.shortening_velocity
+        >>> damper = ForceActuator(damping_force, pathway)
+
+        Again, the forces produces by the damper can be generated by calling
+        the ``compute_loads`` method.
+
+        >>> damper.compute_loads()
+        [(pA, c*q(t)**2*Derivative(q(t), t)/q(t)**2*N.x),
+         (pB, - c*q(t)**2*Derivative(q(t), t)/q(t)**2*N.x)]
+
+        """
+        return self.pathway.compute_loads(self.force)
```

Comment:
I'm still considering whether it wouldn't be better to let `Pathway` compute the force unit vectors and having the multiplication with the force in `Actuator`, such that force itself is decoupled from the `Pathway`.

Reply:
This could well be a worthwhile refactor. However, can I please request that we hold off on this until after I've submitted the PR for wrapping surfaces and wrapping pathways? These significantly complicate the load calculation and I think the structure of this interface will become clearer when we can see more complex cases.